src/packages.builds

```diff
@@ -29,6 +29,9 @@
       to this build if they contain unique 2.2 features. If they are added here you need to ensure their
       versions are bumped higher then what has shipped in release/2.1.
     -->
+    <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.Windows.Compatibility\Microsoft.Windows.Compatibility.builds">
```

Comment:
Perhaps for these case we can delete the Private packages so that they're not built and not published either.